Makefile

```diff
@@ -19,14 +19,19 @@ VERSION ?= v0.3.3
 GOTARGET = github.com/heptio/$(PROJECT)
 OUTPUT_DIR = $(ROOT_DIR)/_output
 BIN_DIR = $(OUTPUT_DIR)/bin
+GIT_SHA=$(shell git rev-parse --short HEAD)
```

Comment:
I wasn't expecting to see the git bits in here, but if you want to kill 2 birds with 1 stone, we can do that...